packages/@lwc/integration-karma/scripts/karma-configs/test/sauce.js

```diff
@@ -51,26 +51,25 @@ const SAUCE_BROWSERS = [
 
     // Compat browsers
     {
-        label: 'sl_ie11',
-        browserName: 'internet explorer',
-        version: '11',
+        label: 'sl_edge_compat',
+        browserName: 'MicrosoftEdge',
+        version: '18',
         compat: true,
```

Comment:
Is `compat` field needed now?

Reply:
See #2948. Yes, because I'm still using "compat" to test legacy browsers in Karma. (I.e. changing the meaning of "compat" for those tests.)